### ISSUE TITLE
Add episode on accessibility

### ISSUE THREAD

**Issue by ndporter** [carpentries/maintainer-onboarding#111]
Create an additional episode in the maintainer onboarding specifically concerning how to ensure lesson webpages are accessible and consistent with international guidelines.

I recommend using [WCAG 2.1 AA](https://www.w3.org/TR/WCAG21/), which is consistent with the [new US rule](https://www.federalregister.gov/documents/2024/04/24/2024-07758/nondiscrimination-on-the-basis-of-disability-accessibility-of-web-information-and-services-of-state) for public-facing communication and currently the near-term aspirational standard internationally.

Many WCAG guidelines are built into workbench, but others (alt-text, link text, heading names, etc) rely on lesson developers and maintainers to implement.

The episode may include links to stable widely-recognized accessibility checker tools, or the process of preparing this may discover Github actions that can be integrated to improve automated checking for accessibility.

Additionally, we should check for any other regional or national rules/guidelines that should be considered to ensure lessons are as internationally compliant as possible.

I can start working on this in Q1 2025, but I'd welcome others who want to join me (or take leadership).

**Comment by ErinBecker** [comment]
@josenino95 - I know you were doing some work recently with @froggleston on accessibility updates to The Workbench. Are there elements of that work that address some of this comment (i.e. that move more of the accessibility checks onto the infrastructure)? Is there information we should add to the onboarding for new Maintainers?

**Comment by josenino95** [comment]
@ErinBecker - We worked on very ad hoc updates, issues that came up to me personally on lessons hosted on our library github org, but by fixing them, the entire workbench infrastructure was benefited, at least on WCAG compliance.  

I don't think that accessibility checks have been moved to the infrastructure, at least in this update.

ADA Title II requirements to comply with WCAG standards are only for US public organizations (I think), so I'm not sure if the global Carpentries would benefit on knowing about this specifically.

If it were up to me, I would add a callout on lesson-infrastructure-wide improvements, for example in terms of accessibility, and basically say who they should contact and via which channel to get that fixed/improved for all lessons. So I wouldn't add something about accessibility checkers for maintainers, **but** probably important for them is to know the basics of accessibility for their lessons and that they should check that minimum standards are put in place, for example alt-text for images. I think the workbench already does some warnings on this, or at least that is what I've seen when building a site locally

**Comment by ndporter** [comment]
@ErinBecker agreed with @josenino95 generally. I think for infrastructure, we can (but don't have to) highlight features of the infrastructure that do the accessibility work for us on things like contrast and page structure.

The areas maintainers may want to know are:
1. Using subheadings rather than text formatting to mark off sections of text
2. Adding useful Alt-text for images (in MD and RMD)
3. Best practices for labeling embedded links (e.g. put the link behind meaningful text, not "here" or as a plaintext URL)

@ErinBecker If you want, I can put together some accessibility tips for maintainers with links to relevant sections of the CLDT when available), though it will help me get to it quicker if you point me to where you think they would fit best
